### ISSUE TITLE
Suggestion : Combine the other git projects into a single repo

### ISSUE THREAD

**Issue by yarivkenan** [Soluto/congo-core#1]
It makes it easier to contribute.

**Comment by guysegal** [comment]
Thanks for the suggestion! We tried a single repo once and it was messy and hard to follow so we thought to create one repo for the core components (for all languages) and repo per plugin

**Comment by yarivkenan** [comment]
Thanks for answering, It will be a great experiment to see this feature per repo attitude pros through the project progress.
